src/components/ui/Chat/Message/Styled.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import styled from 'styled-components';
+
+import { baseStyles, baseSpacing, BaseProps } from '../../Base';
+import { MessageProps } from '../Message';
+
+interface StyledMessageProps extends MessageProps {}
+
+export const StyledMessage = styled.div<StyledMessageProps>`
+  display: flex;
+  flex-direction: row;
+  margin: 0.5rem 0;
+  font-size: 0.65rem;
+  margin-left: 1rem;
+  align-items: center;
+
+  ${baseSpacing}
+  ${baseStyles}
+`;
+
+export const StyledMessageContent = styled.div<StyledMessageProps>`
+  background-color: ${props => props.theme.chatMessage[props.variant].bgd};
+  padding: 0.625rem 1rem;
+  border-radius: 4px;
+  width: fit-content;
+  color: ${props => props.theme.chatMessage[props.variant].fontColor};
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.1);
+  line-height: 20px;
+  width: fit-content;
+  max-width: 70.6%;
+  font-size: 0.875rem;
+
+  & .ch-sender-name {
+    font-weight: bold;
+    margin-bottom: 0.5rem;
+  }
+
+  & svg {
+    position: absolute;
+    margin-top: -3px;
+    margin-left: -20px;
+
+    & g g g {
```

Comment:
That's quite an unusual selector, and possibly not very efficient. Why was it written this way?

Reply:
This caret is an Svg component that is constructed this way. The most inner <g> fill prop is what colors the icon.

Reply:
Gotcha. In cases like these, I tend to add something like a `ClassName` to the element I want to target so that the browser has to do less work to match the element with the style.  It's a small thing, but tends to add up when you have a lot of selectors like this.